src/js/containers/SignUpPage/SignUpPage.js

```diff
@@ -75,7 +75,8 @@ class SignUpPage extends Component {
         );
     }
 
-    handleSignUpSubmit() {
+    handleSignUpSubmit(e) {
+        e.preventDefault()
```

Comment:
The title is misleading. My fault. `e.preventDefault()` doesn't set Access-Control-Allow-Origin header. It just resets the actions to be programmed from start.


> The answer had nothing to do with CORS. The request was cancelled because Chrome had navigated away from the page in the middle of the request. This was happening because event.preventDefault() had not been called in one of the form submit handlers. It seems Chrome handles this differently than other browsers

http://stackoverflow.com/questions/34438540/react-redux-super-agent-first-call-gets-terminated